pages/about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/home.css" />
+    <link rel="stylesheet" href="../styles/about.css" />
+    <title>Friki Corner 2023</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="desktop flexrowDesk navsup">
+          <img src="../Img/FaceIcon.png" alt="Facebook">
+          <img src="../Img/TwitterIcon.png" alt="Twitter">
+          <ul class="ul flexrow">
+            <li>Logout</li>
+            <li>MyPage</li>
+            <li>Mexico</li>
+          </ul>
+        </div>
+        <div class="desktop flexrowDesk navdesk">
+          <a href="../index.html"><img src="../Img/logo.png" alt="Friki Corner"></a>
+          <ul class="ul flexrow">
+            <li><a href="../index.html">Home</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li><button>Friki Corner</button></li>
+          </ul>
+        </div>
+        <div class="mobmenu flexrow">
+          <div class="mobile">
+            <img id="hambur" src="../Img/mobile_menu.png" alt="Menu icon" />
+          </div>
+        </div>
+      </nav>
+      <div id="menu" class="menu">
+        <div class="flexcolum">
+          <img id="close" src="../Img/CloseIcon.png" alt="Close">
+          <ul class="ul flexcolum">
+            <li><a href="../index.html">Home</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+          </ul>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <section class="heading flexcolum">
+        <h3 class="margintop fontsec mobile">"Hello frikies arround the world!"</h3>
+        <h1 class="margintop fontitul">Friki Corner <br />2023</h1>
+        <div class="linea desktop display"></div>
+        <p class="description db">
+          The Friki Corner give the opportunity to enjoy the things you love and enjoy to do with
+          people with the same likes and also bring the opportunity to meet your idols in the comic
+          world.
+        </p>
+        <p class="contact">
+          Please contact us per email for any further questions about Friki Corner 2023!<br />
+          <a href="#">FCconvention@FrikiCorner.org</a>
+        </p>
+      </section>
```

Comment:
Please do your best to stick to the given desing  this sections hight and the width that the items take are not the same as in the template 

![image (8)](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/59707859/6ef7b1c8-5918-4776-a288-4c958e55ba38)